app/frontend/src/components/Navigation/Navigation.tsx

```diff
@@ -1,68 +1,60 @@
 import {
   AppBar,
+  Drawer,
   Grid,
   Hidden,
+  IconButton,
+  List,
+  ListItem,
   makeStyles,
   Toolbar,
   Typography,
 } from "@material-ui/core";
-
-import { useAuthContext } from "../../features/auth/AuthProvider";
-import BugReport from "../../features/BugReport";
-import SearchBox from "../../features/search/SearchBox";
+import { CloseIcon, MenuIcon } from "components/Icons";
+import { useAuthContext } from "features/auth/AuthProvider";
+import BugReport from "features/BugReport";
+import SearchBox from "features/search/SearchBox";
+import React from "react";
 import {
-  connectionsRoute,
+  communityRoute,
+  eventRoute,
   logoutRoute,
-  mapRoute,
   messagesRoute,
   profileRoute,
-} from "../../routes";
-import {
-  CommunityIcon,
-  CrossIcon,
-  EmailIcon,
-  LocationIcon,
-  PeopleIcon,
-  PersonIcon,
-} from "../Icons";
+} from "routes";
+
+import { COUCHERS, LOG_OUT } from "../../constants";
 import NavButton from "./NavButton";
 
 const menu = [
   {
-    icon: <CommunityIcon fontSize="inherit" />,
     name: "Dashboard",
     route: "/",
   },
   {
-    icon: <EmailIcon fontSize="inherit" />,
+    name: "Events",
+    route: eventRoute,
+  },
+  {
     name: "Messages",
     route: messagesRoute,
   },
   {
-    icon: <LocationIcon fontSize="inherit" />,
-    name: "Map",
-    route: mapRoute,
+    name: "Communities",
+    route: communityRoute,
```

Comment:
Should we leave this as the map until communities are in?

Reply:
We're going to have to change it later, so might as well just change it now?